sceptre/plan/plan.py

```diff
@@ -34,9 +34,8 @@ def __init__(self, context):
         self.launch_order = None
 
         config_reader = ConfigReader(context)
-        all_stacks, command_stacks = config_reader.construct_stacks()
-        self.graph = StackGraph(all_stacks)
```

Comment:
I think you can actually iterate on `self.graph` to iterate all stacks, just in order of their dependencies. You might not actually need `self.all_stacks`. You should verify this, but I _think_ you could get away with not actually making any changes here.

Reply:
@jfalkenstein I can't see how, what would be the benefit?

Reply:
I don't understand your statement. My point is that I think you can iterate through the `graph` attribute instead of changing this code to expose `all_stacks`. In other words, I _think_ this change was unnecessary.

Reply:
@jfalkenstein Yes and from reading the code it's not obvious to me how I can iterate through `StackGraph` without adding a method to that class for the purpose of iterating.